<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Imc</class>
 <widget class="QMainWindow" name="Imc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RESULTADO</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/recuso/iconos/mas.png</normaloff>:/recuso/iconos/mas.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>311</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(167, 173, 255);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>101</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>Peso Actual:</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <widget class="QLabel" name="outPesoA">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>91</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>101</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>Peso Mínimo: </string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="outPesoMin">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_5" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>71</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>Peso Máximo:</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_9" native="true">
          <widget class="QLabel" name="outPesoMax">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>121</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_14" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QWidget" name="widget_16" native="true">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>51</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>IMC:</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_17" native="true">
          <widget class="QLabel" name="outIMC">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_10" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>171</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWidget" name="widget_11" native="true">
       <property name="styleSheet">
        <string notr="true">image: url(:/recuso/iconos/bascula.png);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_12" native="true">
       <property name="styleSheet">
        <string notr="true">image: url(:/recuso/iconos/escala.png);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_13" native="true">
       <property name="styleSheet">
        <string notr="true">image: url(:/recuso/iconos/levantamiento-de-pesas.png);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_15" native="true">
       <widget class="QLabel" name="outSalud">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>565</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
